plugins/snowflake/dbt/include/snowflake/macros/catalog.sql

```diff
@@ -24,7 +24,12 @@
               'Approximate Size' as "stats:bytes:label",
               bytes as "stats:bytes:value",
               'Approximate size of the table as reported by Snowflake' as "stats:bytes:description",
-              (bytes is not null) as "stats:bytes:include"
+              (bytes is not null) as "stats:bytes:include",
+
+              'Last Modified' as "stats:last_modified:label",
+              to_varchar(last_altered, 'mon dd, yyyy-HH24:MI:SS') as "stats:last_modified:value",
+              'The timestamp for last update/change' as "stats:last_modified:description",
+              (last_altered is not null) as "stats:last_modified:include"
```

Comment:
The last-modified timestamp is a little less intuitive for views vs. tables. Views will include most recent data, whether or not the view definition has been modified recently. So really, this value just indicates the last time the view **definition** _potentially_ changed, but says nothing about the underlying data.

I'd expect folks with database experience to be comfortable with that distinction, but that's not a background we can presume all dbt-docs viewers to have.

Additionally, views haven't historically had other stats, so this value is presented alone without context. (Hence the failing integration tests, where views are not expected to have stats.)

What do you think about adjusting this to
```suggestion
              (last_altered is not null and table_type = "BASE TABLE") as "stats:last_modified:include"
```